cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/split1/FaceSuite.scala

```diff
@@ -265,17 +270,35 @@ class IdentifyFacesSuite extends TransformerFuzzing[IdentifyFaces] with Cognitiv
     assert(caught.getMessage.contains("Missing required params"))
     assert(caught.getMessage.contains("faceIds"))
   }
+  override def testObjects(): Seq[TestObject[IdentifyFaces]] = Seq(new TestObject(id, df))
 
-  // Utility entrypoint, remove ignore to clean up manually
-  ignore("Clean up all personGroups") {
+  def cleanOldGroups(): Unit = {
+    val twoDaysAgo = LocalDateTime.now().minusDays(2)
+    PersonGroup.list().foreach { pgi =>
+      try {
+        val pgDateString = pgi.personGroupId.replaceFirst("group", "")
+        val pgDate = LocalDateTime.parse(pgDateString, DateTimeFormatter.ofPattern(timeFormat))
+        if (twoDaysAgo.compareTo(pgDate) <= 0) {
```

Comment:
I htink you can use something a little more readable: pgDate.isBefore(twoDaysAgo)